test/integration/script_lint_test.go

```diff
@@ -0,0 +1,100 @@
+package integration
```

Comment:
what does this do?

Reply:
It performs a "lint" test of all scripts in the helm charts.

The "lint" test is performed by running each found script with the '-n' option.